js/src/components/account-card/index.scss

```diff
@@ -45,8 +45,8 @@
 		// but the default link style has a smaller size that doesn't meet
 		// the visual design needs. So we override its size here.
 		> .components-button.is-link {
-			min-height: 36px;
-			padding: 6px 12px;
+			min-height: $gla-size-control-height;
+			padding: $gla-grid-unit $grid-unit-15;
```

Comment:
🧵 
The variables for grid units declared in Gutenberg should be useful enough. I think GLA doesn't need to declare another list of grid units itself. For the 6px value, it could be `$grid-unit-15 / 2`.

Reply:
Addressed in last commits